Dangerfile

```diff
@@ -10,7 +10,8 @@ end
 
 require 'json'
 
-rspec_files = Dir[File.join(ENV["CIRCLE_ARTIFACTS"], "rspec_logs_*.json")]
+containing_dir = ENV["CIRCLE_ARTIFACTS"] || "." # for local testing
```

Comment:
could the setting to "."by default cause unexpected issues (e.g.) if you run inside a different CI environment, etc.

Also, I noticed that I tried to use circleci to build my own branch of fastlane and it started to send information to some connected services. I.e. it doesn't differentiate YOUR circleci config, with mine.

At some point we should perhaps depend on a more explicit environment config.